hardhat_scripts/staging_redeem_queue_migration/hoodi/03_verify_queue.ts

```
@@ -0,0 +1,395 @@
+  // invisible here and the reconciliation is reported but not enforced.
+  const history = historyProvider(hre.ethers);
+  const head = Math.min(block, await history.getBlockNumber());
+  const claims = await claimsSince(history, generatedAtBlock, head);
+  const unreconciled: string[] = [];
+  for (const c of comparisons) {
+    const claimed = claims.get(c.id) ?? { lsETH: hre.ethers.constants.Zero, eth: hre.ethers.constants.Zero };
+    if (!claimed.lsETH.eq(c.claimedLsETH) || !claimed.eth.eq(c.claimedEth)) unreconciled.push(`${c.id}`);
+  }
+
+  console.log("queue vs the provided payload");
+  console.log(`  records matching exactly              : ${comparisons.length - claimedSince.length - mismatched.length}/${provided.length}`);
```

ESLint

[error] Replace ``··records·matching·exactly··············:·${comparisons.length·-·claimedSince.length·-·mismatched.length}/${provided.length}`` with `⏎····`··records·matching·exactly··············:·${comparisons.length·-·claimedSince.length·-·mismatched.length}/${⏎······provided.length⏎····}`⏎··`

(prettier/prettier)